tests/Fixtures/inc/Engine/Optimization/Minify/CSS/AdminSubscriber/cleanMinify.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Test Data for Cache Dynamic Resource.
+ */
+
+return [
+	'vfs_dir'   => 'wp-content/cache/min/',
+	// Virtual filesystem structure.
+	'structure' => [
+		'wp-content' => [
+			'cache' => [
+				'min' => [
+					'1'         => [
+						'fa2965d41f1515951de523cecb81f85e.css' => '',
+						'2n7x3vd41f1515951de523cecb81f85e.css' => '',
+					],
+					'3rd-party' => [
+						'2n7x3vd41f1515951de523cecb81f85e.css' => '',
+						'fa2965d41f1515951de523cecb81f85e.css' => '',
+					],
+				],
+			],
+		],
+	],
+	// Default settings.
+	'settings'  => [
+		'minify_css'  => false,
+		'exclude_css' => [],
+		'cdn'         => false,
+		'cdn_cnames'  => [],
+	],
+	'test_data' => [
+		'shouldNotCleanMinify'             => [
+			'value'     => [
+				'minify_css'  => false,
+				'exclude_css' => [],
+				'cdn'         => false,
+				'cdn_cnames'  => [],
+			],
+			'should_run' => false,
+		],
+		'shouldCleanMinifyCSS'             => [
+			'value'     => [
+				'minify_css'  => true,
+				'exclude_css' => [],
+				'cdn'         => false,
+				'cdn_cnames'  => [],
+			],
+			'should_run' => true,
+		],
+		'shouldCleanMinifyExcludeCSS'      => [
+			'value'     => [
+				'minify_css'  => true,
+				'exclude_css' => [ '.css_to_exclude' ],
```

Comment:
This is not blocking, but it might become an unexpected issue later: this field expects relative path(s) to file(s), not CSS content

Reply:
Fixed. Thank you for spotting this issue :)